source/includes/note-collections-in-mixed-sync.rst

```diff
@@ -0,0 +1,23 @@
+.. note:: Collections in Mixed fields supported in Apps created after May 22, 2024
+
+    App Services Apps created *after* May 22, 2024 can store collections
```

Comment:
Thank you for updating this! I think technically this was out in prod on May 29th but I'll defer to @tkaye407 since May 22nd was the official release date